spec/models/location_spec.rb

```diff
@@ -19,6 +19,11 @@
     it 'has nooks and reservations' do
       expect(location).to respond_to :nooks, :reservations
     end
+
+    it 'must have a name' do
+      location.save
+      expect(location.errors.full_messages).to include("Name can't be blank")
```

Comment:
Just noticed that even in the controller the messages are hard coded. Approving!